### DISCUSSION TITLE
Login Vaultwarden Webvault - WebAuthN/Passkey in Bitwarden app not working with 1.32.0

### DISCUSSION THREAD

**Discussion by belerovon** [dani-garcia/vaultwarden#4854] [Q&A]
### Subject of the issue
As an 2nd Factor I used WebAuthN and Configured it in my Bitwarden Plugin.
When Logging in to my Webvault, the Plugin offers me normaly my PassKey.

I upgraded from 1.31.0 to 1.32.0. 
Now the Bitwarden App (windows, mac, safari, Firewfox or chrome) not detect the Passkey.
On iOS it still works.

When downgrading to 1.31.0 the Passjey 

### Deployment environment

* Install method: Docker on Debian with Apache Reverse Proxy

* Clients used: Mac / Windows - with Firefox, Safari and Chrome Browser

* Reverse proxy and version: apache2

* MySQL/MariaDB or PostgreSQL version: PostgreSQL 

* Other relevant details:

### Steps to reproduce
Upgrade to 1.32.0. Login to Webvault. 
On second factor only the attached Hardware Tokens work, not the PassKey in Bitwarden Plugin

### Expected behaviour
Bitwarden Plugin popup and present me available passkeys

### Actual behaviour
System Window Pop up and offers me physical Tokens

**Comment by BlackDex** [comment]
If you try to login using passkeys into your Vaultwarden account, that isn't supported, so that would be correct and not an issue.

**Comment by BlackDex** [comment]
Also, i just tried to use the Bitwarden Firefox Extension (v2024.7.1) and that works just fine as a WebauthN device for me.
Also in combination with a physical hardware token also linked.
